po/de.po

```diff
@@ -0,0 +1,695 @@
+msgstr "Konnte Einlade-Link nicht erstellen: {}"
+
+#. "description" is hard-coded in bitlbee as possible field for chat topic
+#: ../td-client.cpp:2130
+msgid "Description"
+msgstr "Beschreibung"
+
+#: ../td-client.cpp:2179 ../td-client.cpp:2195 ../td-client.cpp:2214
+#: ../td-client.cpp:2234 ../td-client.cpp:2237 ../td-client.cpp:2242
+#: ../tdlib-purple.cpp:822
+msgid "Two-factor authentication"
+msgstr "Zwei-Faktor-Authentifizierung"
+
+#: ../td-client.cpp:2180
+msgid "Enter verification code received in the e-mail"
+msgstr "Bitte Bestätigungs-Code aus E-Mail eingeben"
+
+#: ../td-client.cpp:2196
+msgid "Password set"
+msgstr "Passwort gesetzt"
+
+#: ../td-client.cpp:2196
+msgid "Password cleared"
+msgstr "Passwort entfernt"
+
+#: ../td-client.cpp:2197
+msgid "Recovery e-mail is configured"
+msgstr "Wiederherstellungs-E-Mail gesetzt"
+
+#: ../td-client.cpp:2198
+msgid "No recovery e-mail configured"
+msgstr "Keine Wiederherstellungs-E-Mail gesetzt"
+
+#: ../td-client.cpp:2206
+msgid "Code sent to {} (length: {})"
+msgstr "Code gesendet an {} (Länge: {})"
+
+#: ../td-client.cpp:2214
+msgid "Failed to set password"
+msgstr "Konnte Passwort nicht setzen"
+
+#: ../td-client.cpp:2232
+msgid "E-mail address: {}"
+msgstr "E-Mail Adresse: {}"
+
+#: ../td-client.cpp:2235
+msgid "For some reason, new confirmation code was sent"
+msgstr "Seltsamerweise wurde ein neuer Bestätigungs-Code gesendet"
+
+#: ../td-client.cpp:2237
+msgid "Looks like the code was wrong"
+msgstr "Der Code war falsch"
+
+#: ../td-client.cpp:2242
+msgid "Failed to verify recovery e-mail"
+msgstr "Konnte Wiederherstellungs-E-Mail nicht bestätigen"
+
+#: ../td-client.cpp:2313
+msgid "Unable to send message: The message is too large."
+msgstr "Konnte Nachricht nicht senden: Zu groß."
+
+#: ../td-client.cpp:2315
+msgid "Unable to send message."
+msgstr "Konnte Nachricht nicht senden."
+
+#: ../td-client.cpp:2317
+msgid "Failed to open chat: {}"
+msgstr "Konnte Chat nicht öffnen: {}"
+
+#: ../tdlib-purple.cpp:33
+msgid "now"
+msgstr "jetzt"
+
+#: ../tdlib-purple.cpp:40
+msgid "recently"
+msgstr "kürzlich"
+
+#: ../tdlib-purple.cpp:42
+msgid "last week"
+msgstr "letzte Woche"
+
+#: ../tdlib-purple.cpp:44
+msgid "last month"
+msgstr "letzten Monat"
+
+#: ../tdlib-purple.cpp:61 ../tdlib-purple.cpp:279
+msgid "Last online"
+msgstr "Zuletzt online"
+
+#: ../tdlib-purple.cpp:123 ../tdlib-purple.cpp:129
+msgid "Leaving group"
+msgstr "Gruppe verlassen"
+
+#: ../tdlib-purple.cpp:124
+msgid "Confirm deleting group"
+msgstr "Löschen der Gruppe bestätigen"
+
+#: ../tdlib-purple.cpp:125
+msgid "Leaving basic group you created will delete the group. Continue?"
+msgstr ""
+"Eine „normale“ Gruppe die man erstellt hat zu verlassen löscht die Gruppe. "
+"Fortfahren?"
+
+#: ../tdlib-purple.cpp:127 ../tdlib-purple.cpp:132 ../tdlib-purple.cpp:156
+msgid "_Yes"
+msgstr "_Ja"
+
+#: ../tdlib-purple.cpp:127 ../tdlib-purple.cpp:132 ../tdlib-purple.cpp:156
+msgid "_No"
+msgstr "_Nein"
+
```

Comment:
So if I switch to using purple_request_yes_no and remove these from here (as they wouldn't have appeared here had I used purple_request_yes_no to begin with), then the buttons become untranslated. So I'll keep everything as it is, and probably replace purple_request_accept_cancel with purple_request_input as well.

Reply:
Sounds like a good idea.